src/tests/ftest/util/server_utils.py

```diff
@@ -30,14 +30,16 @@
 import subprocess
 import json
 import re
-import resource
 import signal
-import fcntl
 import errno
 import yaml
+import getpass
 
-from avocado.utils import genio
-from general_utils import pcmd
+from general_utils import pcmd, check_file_exists
+from command_utils import JobManager, DaosCommand, Orterun
```

Comment:
(pylint-import-error) Unable to import 'command_utils' (invalid syntax (<string>, line 333))
(pylint-unused-import) Unused JobManager imported from command_utils